frigate/models.py

```diff
@@ -1,14 +1,15 @@
 from numpy import unique
 from peewee import *
 from playhouse.sqlite_ext import *
+from playhouse.mysql_ext import *
 
 
 class Event(Model):
     id = CharField(null=False, primary_key=True, max_length=30)
     label = CharField(index=True, max_length=20)
     camera = CharField(index=True, max_length=20)
     start_time = DateTimeField()
-    end_time = DateTimeField()
+    end_time = DateTimeField(null=True)
```

Comment:
Just curious, why is this needed?

Reply:
I found when testing, pymysql doesnt set DEFAULT NULL without specifying this.
frigate/events.py tries to set this as None when starting an event.